Scripts/Web_Scrappers/Shopify_Web_Scraper/shopify.py

```diff
@@ -0,0 +1,234 @@
+import argparse
+import csv
+import json
```

Comment:
import specific method from module
```suggestion
from json import loads
```